src/System.Threading.Overlapped/tests/System.Threading.Overlapped.Tests.csproj

```diff
@@ -26,6 +26,9 @@
     <Compile Include="ThreadPoolBoundHandle_DisposeTests.cs" />
     <Compile Include="ThreadPoolBoundHandle_BindHandleTests.cs" />
     <Compile Include="OverlappedTests.cs" />
+    <Compile Include="$(CommonTestPath)\System\IO\FileCleanupTestBase.cs">
+      <Link>Common\System\IO\FileCleanupTestBase.cs</Link>
```

Comment:
With @tarekgh's change this shouldn't be needed as he moved this class to the new TestUtils project in: https://github.com/dotnet/corefx/pull/21014